### DISCUSSION TITLE
Log retention policy problem

### DISCUSSION THREAD

**Discussion by mafer-22** [Security-Onion-Solutions/securityonion#12252] [Unsupported Versions]
Hi, i have problems to make SO set the retention policy to 90 days, i already updated the log_size_limit to 990 Gb, and i also updated the life cycle policy at kibana, but is not working it keeps errasing logs older than 30 days, can someone help me with this please? I have the 2.3 version
![Limite de espacio despues](https://github.com/Security-Onion-Solutions/securityonion/assets/127130790/a06639f4-6782-4009-98bb-982dd3349d3b)
![Periodo de retencion después](https://github.com/Security-Onion-Solutions/securityonion/assets/127130790/4c5d3f84-994b-4a53-8372-1a19b4d4bcfd)

**Comment by cm-ops** [comment]
In 2.3, Curator handles that. You would make your changes in your `global.sls`.

**Reply by mafer-22** [reply]
Thank you, checking the global.sls file, i found that all my index settings looks like this 
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/127130790/9c8513bb-6517-42ae-a253-a7707a8bdcf9)

So what i understand is that i have 365 days before curator starts deleting logs but all the logs are getting deleted after 30 days

**Reply by mafer-22** [reply]
Should i add another configuration line in this file?

**Reply by cm-ops** [reply]
What is the output of `du -hd1 /nsm`? There is also a cron that will delete based on disk utilization.
